### PR TITLE
Gtc integration python path

### PR DESCRIPTION
**Pull request by DropD**
## Description

The python IR path

## Requirements

Before submitting this PR, please make sure:

- [ ] The code builds cleanly without new errors or warnings
- [ ] The code passes all the existing tests
- [ ] If this PR adds a new feature, new tests have been added to test these
new features
- [ ] All relevant documentation has been updated or added

Additionally, if this PR contains code authored by new contributors:

- [ ] All the authors are covered by a valid contributor assignment agreement,
signed by the employer if needed, provided to ETH Zurich
- [ ] The names of all the new contributors have been added to an updated
version of the AUTHORS.rst file included in the PR

### CODE REVIEW

File: tests/test_unittest/test_gtc_backend/test_fields_metadata_pass.py [hunk 1]
```diff
@@ -2,9 +2,8 @@
```
Comment (havogt) [inline tests/test_unittest/test_gtc_backend/test_fields_metadata_pass.py:2]: should go to `test_gtc2`?

Comment (havogt) [reply tests/test_unittest/test_gtc_backend/test_fields_metadata_pass.py:2]: and some others below as well

**Review by havogt — COMMENTED** [review]
Maybe we should introduce subdirectories for each gtc2 backend (i.e. lowest level IR + generators). What do you think?

**Comment by havogt** [comment]
I don't remember exactly, but wasn't our last idea to put gtc inside of the gt4py directory?

File: tests/test_unittest/test_gtc_backend/test_fields_metadata_pass.py (hunk 1 above)
Comment (DropD) [reply tests/test_unittest/test_gtc_backend/test_fields_metadata_pass.py:2]: > Maybe we should introduce subdirectories for each gtc2 backend (i.e. lowest level IR + generators). What do you think?

 I think that makes sense

**Comment by DropD** [comment]
> I don't remember exactly, but wasn't our last idea to put gtc inside of the gt4py directory?

not sure actually

**Review by havogt — APPROVED** [review]

**Pull request merged**